app/controllers/qa/issues_controller.rb

```diff
@@ -20,11 +20,19 @@ def edit
   end
 
   def update
-    @issues = current_project.issues.ready_for_review.where(id: params[:ids])
+    @issues = current_project.issues.where(id: params[:ids])
```

Comment:
the use of :ids instead of :id, breaks the convention we use elsewhere in #update actions. Why is that the right call?